app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
+  },
+  "step3HardwareWallet": {
+    "message": "3. Empezar a usar dApps y más!"
+  },
+  "step3HardwareWalletMsg": {
+    "message": "Usa tu cuenta física igual que harías con cualquier cuenta de Ethereum. Regístrate con dApps, manda Eth, compra y almacena tokens de ERC20 y otros tokens no-fungibles, como CryptoKitties."
+  },
+  "token": {
+    "message": "Token"
+  },
+  "transaction": {
+    "message": "transacción"
+  },
+  "transactionConfirmed": {
+    "message": "Se confirmó la transacción a $2."
+  },
+  "transactionCreated": {
+    "message": "Se creó una transacción con un valor de $1, a $2."
+  },
+  "transactionWithNonce": {
+    "message": "Transacción $1"
+  },
+  "transactionDropped": {
+    "message": "Transaction se cayó en $2."
+  },
+  "transactionSubmitted": {
+    "message": "Se propuso la transacción con una comisión de gas de $1, en $2."
+  },
+  "transactionResubmitted": {
+    "message": "Se volvió a proponer la transacción, aumentando la comisión de gas a $1, en $2"
+  },
+  "transactionUpdated": {
+    "message": "Se actualizó la transacción en $2."
+  },
+  "transactionUpdatedGas": {
+    "message": "Se actualizó la transacción, con una comisión de gas de $1 en $2."
+  },
+  "transactionErrored": {
+    "message": "La transacción tuvo un error."
+  },
+  "transactionCancelAttempted": {
+    "message": "Se intentó cancelar la transacción con una comisión de gas de $1, en $2"
+  },
+  "transactionCancelSuccess": {
+    "message": "La transacción se canceló con éxito en $2"
+  },
+  "transactionError": {
+    "message": "Error en transacción. Se produjo una excepción en el código del contrato."
+  },
+  "transactionErrorNoContract": {
+    "message": "Intentando llamar una función en una dirección que no es del contrato."
+  },
+  "transactionFee": {
+    "message": "Comisión de la transacción"
+  },
+  "transactionTime": {
+    "message": "Tiempo de Transacción"
+  },
+  "transfer": {
+    "message": "Traspasar"
+  },
+  "transferFrom": {
+    "message": "Traspasar De"
+  },
+  "trezorHardwareWallet": {
+    "message": "Monedero Físico TREZOR"
+  },
+  "tryAgain": {
+    "message": "Vuelve a intentar"
+  },
+  "units": {
+    "message": "unidades"
+  },
+  "unknownQrCode": {
+    "message": "Error: No pudimos identificar ese código QR"
+  },
+  "unknownCameraErrorTitle": {
+    "message": "Uuups! Algo se estropeó..."
+  },
+  "unknownCameraError": {
+    "message": "Hubo un error al intentar acceder a la cámara. Por favor, vuelve a intentar..."
+  },
+  "unlock": {
+    "message": "Desbloquear"
+  },
+  "unlockMessage": {
+    "message": "Te espera toda la red descentralizada"
+  },
+  "updatedWithDate": {
+    "message": "Actualizado $1"
+  },
+  "viewOnEtherscan": {
+    "message": "Ver en Etherscan"
+  },
+  "welcomeBack": {
+    "message": "¡Bienvenido de nuevo!"
+  },
+  "yesLetsTry": {
+    "message": "Sí pues, vamos a probar"
```

Comment:
The use of "pues" could be very location bias, I'm not sure how many Spanish speaker countries use it

Reply:
what you guys think about, "sí, probemos"?

Reply:
Copy that. Will make the change.